pinot-segment-spi/src/main/java/org/apache/pinot/segment/spi/Constants.java

```diff
@@ -29,6 +29,8 @@ private Constants() {
   public static final String HLLPLUS_ULL_P_KEY = "p";
   public static final String HLLPLUS_SP_KEY = "sp";
   public static final String CPCSKETCH_LGK_KEY = "lgK";
+  public static final String THETASKETCH_NOMINAL_ENTRIES = "K";
+  public static final String TUPLESKETCH_NOMINAL_ENTRIES = "K";
```

Comment:
Would `nominalEntries` potentially be more user-friendly than `K`? Or maybe we could accept either. Also, I think we could use a single constant for this (`THETA_TUPLE_SKETCH_NOMINAL_ENTRIES`) similar to the `p` key for `HLLPLUS` / `ULL`?

Reply:
The datasketches library code uses K throughout but I agree that it is more user friendly to use your suggestion and it also aligns with the parameter name that is passed to the aggregation function.